package.json

```diff
@@ -15,6 +15,7 @@
     "chalk": "^2.4.1",
     "cli-table": "github:Automattic/cli-table#master",
     "inquirer": "^6.2.0",
+    "joi": "^14.3.1",
```

Comment:
This is what we currently have for all dependencies here.
But I don't especially trust `^`, we should define specific versions.
For me, the benefits of having patchs applied automatically (if every dependencies follows perfectly the semver pattern), is not big enough. Compared to the "risk" of having a patch applied on a release that would break something without knowing it.

Reply:
Yes I 100% agree, this happend because `save-exact=true` was not set in `.npmrc`, this is done now on master